osu.Game/Screens/Play/LocalUserPlayingStates.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Screens.Play
+{
+    public enum LocalUserPlayingStates
```

Comment:
Bit weird for this enum name to be in plural, I'd normally expect `LocalUserPlayingState`...?

Reply:
I originally had it plural to avoid the classic namespace conflict with field (was called `PlayingState`). Can go with non-plural not, sure.